public/.htaccess

```diff
@@ -0,0 +1,5 @@
+RewriteEngine on
+RewriteCond %{REQUEST_FILENAME} !-d
+RewriteCond %{REQUEST_FILENAME} !-f
+
+RewriteRule . index.php [L]
```

Comment:
I think that we should keep it simple enough to allow Equip to run properly, and if someone wants custom adjustments just leave it up to them.